.ackrc

```diff
@@ -1,2 +1,8 @@
 --ignore-directory=docs/_generated
 --ignore-directory=htmlcov
+--ignore-directory=.mypy_cache
+--ignore-directory=.ipynb_checkpoints
+--ignore-directory=userHelp
+--ignore-file=ext:lock
+--ignore-file=ext:log
+--ignore-file=ext:xml
```

Comment:
I had to look it up as we've never touched this file since you created this in PR https://github.com/scikit-hep/pyhf/pull/305, but I guess this is like a dotfile to `ack`. What is this file actually used for though. I guess we need it for something in the tests, but are these additions needed or just recommended by a tool?

Reply:
just to grep for changes, I'll remove it.